tests/test_main.py

```diff
@@ -153,3 +156,36 @@ def test_get_credits_balance_live(auth_live, monkeypatch):
     balance = get_credits_balance()
     assert isinstance(balance, dict)
     assert "balance" in balance
+
+
+def test_get_auth_safely_no_auth(monkeypatch):
+    monkeypatch.setattr(main, "_auth", None)
+    with pytest.raises(ValueError) as excinfo:
+        main.__get_auth_safely()
+    assert "User not authenticated. Call up42.authenticate() first" in str(excinfo.value)
+
+
+def test_get_webhook_events(auth_mock, requests_mock, monkeypatch):
+    monkeypatch.setattr(main, "_auth", auth_mock)
+    url_webhook_events = f"{API_HOST}/webhooks/events"
+    requests_mock.get(
+        url=url_webhook_events,
+        json={
+            "data": ["some event"],
+            "error": {},
+        },
+    )
+    events = main.get_webhook_events()
+    assert isinstance(events, List)
+    assert "some event" in events
+
+
+@pytest.mark.parametrize("return_json", [False, True])
+def test_get_webhooks(auth_mock, monkeypatch, webhooks_mock, return_json):
+    monkeypatch.setattr(main, "_auth", auth_mock)
+    webhooks = main.get_webhooks(return_json=return_json)
+    assert isinstance(webhooks, List)
```

Comment:
We need more strict check here. There are two ways to address:
1. Add `__eq__` implementation with coverage to Webhook using webhookd_id and _info, then this can be simplified to 
```python
assert main.get_webhooks(return_json=return_json) == webhooks_mock.get_webhooks(return_json=return_json)
```
2. Use direct check on properties
```python
    webhooks = main.get_webhooks(return_json=return_json)
    expected = webhooks_mock.get_webhooks(return_json=return_json)
    if return_json:
        assert webhooks == expected
    else:
        for hook, expected_hook in zip(webhooks, expected):
            assert hook.webhook_id == expected_hook.webhook_id
            assert hook._info == expected_hook._info
```